grammars/pl sql (oracle).cson

```diff
@@ -31,6 +31,10 @@ foldingStopMarker: "(?i)^\\s*(end)\\b"
 name: "PL/SQL (Oracle)"
 patterns: [
   {
+    match: "^.*(logger\\.|logger_logs\\.).*$"
```

Comment:
Tested with: `match: "^.*(logger\\.|logger_logs\\.)[^;]+"`

![image](https://cloud.githubusercontent.com/assets/1747643/19788943/6089ac6a-9cf7-11e6-8b72-955393f95591.png)

Can you update with this (Still not perfect, but slight improvement I think, having tested some basic scenarios). If you're too busy, I'll merge and make this change later.